Kudu.Core/Functions/KafkaTriggerKedaAuthProvider.cs

```diff
@@ -76,5 +111,22 @@ internal string getKedaProperty(string triggerBinding)
             }
             return TriggerAuthConstants.KafkaTriggerBindingToKedaProperty.GetValueOrDefault(triggerBinding);
         }
+
+        internal Boolean IsTriggerAuthRequired(IDictionary<string, string> functionBindings, string functionName) {
+
+            if (!functionBindings.ContainsKey(TriggerAuthConstants.KAFKA_TRIGGER_PROTOCOL) || !functionBindings.ContainsKey(TriggerAuthConstants.KAFKA_TRIGGER_AUTH_MODE)) {
+                return false;
+            }
+
+            if (functionBindings[TriggerAuthConstants.KAFKA_TRIGGER_AUTH_MODE].Equals(TriggerAuthConstants.KAFKA_TRIGGER_AUTH_MODE_NOT_SET, StringComparison.OrdinalIgnoreCase) 
+                    || functionBindings[TriggerAuthConstants.KAFKA_TRIGGER_AUTH_MODE].Equals(TriggerAuthConstants.KAFKA_TRIGGER_PROTOCOL_NOT_SET, StringComparison.OrdinalIgnoreCase)) {
+                return false;
+            }
+
+            if (functionBindings[TriggerAuthConstants.KAFKA_TRIGGER_AUTH_MODE].Equals("Gssapi", StringComparison.OrdinalIgnoreCase)) {
+                Console.WriteLine("Gssapi as Authentication Mode is not supported in Keda, function app {0} might not be able to scale", functionName);
```

Comment:
Should we return false here ?

Reply:
We are logging and silently moving on, as keda ahead with handle this case